package.json

```diff
@@ -9,6 +9,10 @@
 	"author": "",
 	"version": "1.0.0",
 	"license": "ISC",
+	"engines": {
+		"node": ">=18.19.0 <19.0.0",
```

Comment:
is there a reason to lock it to ```<19.0.0``` ? I was testing it using ```21.x```.
Should .nvmrc be enough for setting node version? Then we can maybe widen this requirement a bit.

Reply:
If you use nodejs 22 for example to add dependency, it could happen that a version is installed/added to the `package.json` / `package-lock.json` not installable with an
older nodejs/npm version.

The nodejs/npm ecosystem is quit sensible in this area. Thus we have it nailed down to specific nodejs/npm version in the core to avoid such issues here.

And it allows a smaller diverging chain across `.nvmrc` and eventually docker image using (later in runTest.sh` as it enforces to keep them all in the same line.

On the otherhand - I'm not a nodejs ecosystem expert. So no hard opionen on this in general beside opening for missmatch between local and ci runs.